Alignment/CommonAlignmentProducer/python/ALCARECOPromptCalibProdSiPixelAliHLTHGDiMuon_cff.py

```diff
@@ -0,0 +1,139 @@
+import FWCore.ParameterSet.Config as cms
+
+# ------------------------------------------------------------------------------
+# configure a filter to run only on the events selected by TkAlZMuMu AlcaReco
+from HLTrigger.HLTfilters.hltHighLevel_cfi import *
+ALCARECOTkAlZMuMuFilterForSiPixelAliHLT = hltHighLevel.clone(
```

Comment:
Don't you think to clone them from `triggerResultsFilterFromDB`, as modified for other AlCa filters in https://github.com/cms-sw/cmssw/pull/46742? I understand that you probably don't plan to prescale further these paths, but I wonder if the same configurations could be used for all them. Unless there is a real advantage in cloning from `hltHighLevel`

Reply:
Same comment applies to the other similar instances in this PR

Reply:
In this specific instance you have comment upon it's not possible to use `triggerResultsFilterFromDB` because the filtering is not based on a path key retrieved from the event setup but rather on the name of the path `pathALCARECOTkAlHLTTracksZMuMu` which is executed in the step before (the alcareco sample creation: this ensures that the producer is run only in presence of the right input set of tracks).
Elsewhere it might be possible to use it, but there is no advantage in doing to. The input HLT paths are actually already limited in rate (see [CMSHLT-3404](https://its.cern.ch/jira/browse/CMSHLT-3404)), thus prescaling is not desireable - we'll actually be statistically limited.